src/wp-includes/media.php

```diff
@@ -2043,13 +2037,17 @@ function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
  * to remove the attribute or set it to another accepted value.
  *
  * @since 6.1.0
+ * @deprecated 6.4.0 Use wp_get_loading_optimization_attributes()
  *
  * @param string $image   The HTML `img` tag where the attribute should be added.
  * @param string $context Additional context to pass to the filters.
  *
  * @return string Converted `img` tag with `decoding` attribute added.
  */
 function wp_img_tag_add_decoding_attr( $image, $context ) {
```

Comment:
Note that deprecated functions should be moved to the `deprecated.php` file.

However, before you do that, please make sure to revert any changes to the actual function logic (other than the `_deprecated_function()` call), since no changes should be made to deprecated functions unless absolutely necessary. That way, it also won't be difficult to review the changes, as we can assume the function logic is unchanged compared to before the code was moved.

Reply:
I'm leaving this out for now while we check the other changes.